frontend/apps/cloudstorage/webdavapi.lua

```diff
@@ -124,6 +131,7 @@ function WebDavApi:listFolder(address, user, pass, folder_path, folder_mode)
                     text = item_name,
                     url = item_path,
                     type = "file",
+                    filesize = item_filesize
```

Comment:
tonumber is okay with nil, you can get rid of item_filesize

`filesize = tonumber(item_content_length_match),`